setup.py

```diff
@@ -38,14 +38,15 @@ def reqs(*f):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Framework :: Django',
-        'Framework :: Django :: 1.4',
-        'Framework :: Django :: 1.5',
-        'Framework :: Django :: 1.6',
-        'Framework :: Django :: 1.7',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
+        'Framework :: Django :: 1.12',
```

Comment:
These versions are not going to exist so you don't need them in the classifiers.